hpedockerplugin/hpe/hpe_3par_fc.py

```diff
@@ -264,6 +264,18 @@ def _build_initiator_target_map(self, common, connector):
         return target_wwns, init_targ_map, numPaths
 
 
+    def _modify_3par_fibrechan_host(self, common, hostname, wwn):
+        mod_request = {'pathOperation': common.client.HOST_EDIT_ADD,
+                       'FCWWNs': wwn}
+        try:
+            common.client.modifyHost(hostname, mod_request)
+        except hpeexceptions.HTTPConflict as path_conflict:
+            msg = _LE("Modify FC Host %(hostname)s caught "
+                      "HTTP conflict code: %(code)s")
+            LOG.exception(msg,

```

Comment:
when chewing checked exception for logging, which doesnt necessarily warrant stopping the code flow, it makes sense to use "WARN" log level instead of "ERROR"